### PR TITLE
enhance cpu advisor robustness

### PR DESCRIPTION
**Pull request by sun-yuliang**
#### What type of PR is this?
Bug fixes & Enhancements

#### What this PR does / why we need it:
Several fixes and enhancement to improve cpu advisor robustness

### CODE REVIEW

File: pkg/agent/sysadvisor/plugin/qosaware/server/memory_server.go [hunk 1]
```diff
@@ -64,26 +65,26 @@ func (ms *memoryServer) ListAndWatch(_ *advisorsvc.Empty, server advisorsvc.Advi
 	for {
 		select {
 		case <-ms.stopCh:
-			general.Infof("lw stopped because %v stopped", ms.name)
+			klog.Infof("[qosaware-server-memory] lw stopped because %v stopped", ms.name)
```
Comment (waynepeking348) [inline pkg/agent/sysadvisor/plugin/qosaware/server/memory_server.go:68]: why we should switch general log back to klog ...

Comment (sun-yuliang) [reply pkg/agent/sysadvisor/plugin/qosaware/server/memory_server.go:68]: Prefix is needed for more efficient debug; Package name logged by general decreases readability.

Comment (waynepeking348) [reply pkg/agent/sysadvisor/plugin/qosaware/server/memory_server.go:68]: > Prefix is needed for more efficient debug. Package name logged by general decreases readability.

all right... still kind of ugly to have so many [xxx] spread everywhere ... i will not insist, but i have to point this coding style...

Comment (sun-yuliang) [reply pkg/agent/sysadvisor/plugin/qosaware/server/memory_server.go:68]: trade off between quicker bug location by log & slight better code style : )

**Review by waynepeking348 — APPROVED** [review]

**Review by csfldf — APPROVED** [review]

**Pull request merged**